src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -315,6 +323,11 @@
           (swap! state update :server-response assoc :server-error err-text)
           (swap! state update :server-response
                  assoc :billing-projects (map #(% "projectName") projects)))))
+     (endpoints/get-library-groups
+      (fn [{:keys [success? get-parsed-response]}]
+        (if success?
+          (swap! state update :server-response assoc :library-groups (get-parsed-response))
+          (swap! state update :server-response assoc :server-error "Unable to load library groups"))))
```

Comment:
do we need this on the workspace summary tab? If we don't need this until the catalog wizard is opened, could you move this call there? (we can do this as a follow-on PR, but preferably in this sprint). We already make a lot of calls on the workspace summary page, and I'd like to avoid adding more unless we absolutely have to.